templates/dashboard/support.html

```diff
@@ -0,0 +1,101 @@
+            font-weight: 600;
+            margin-bottom: 3px;
+        }
+    </style>
+
+    <script src="{{ url_for('static', filename='node_modules/vue/dist/vue.min.js') }}"></script>
+    <script>
+        $( document ).ready(function() {
+            var app = new Vue({
+                el: '#aliasGroup',
+                data: {
+                    ticketEmail: document.querySelector("input[name=ticketEmail]").value
+                },
+                methods: {
+                    generateRandomAlias: async function(event){
+                        let result = await fetch('/api/alias/random/new', { method: 'POST'})
+                        if(result.ok){
+                            let data = await result.json();
+                            this.ticketEmail = data.alias;
+                        }
+                    }
+                }
+            });
+
+            $('.custom-file input').change(function (e) {
+                var files = [];
+                for (var i = 0; i < $(this)[0].files.length; i++) {
+                    files.push($(this)[0].files[i].name);
+                }
+                $(this).next('.custom-file-label').html(files.join(', '));
+            });
+
+
+        })
+    </script>
+{% endblock %}
+
+{% block default_content %}
+
+    <div class="col pb-3">
+        <!-- Current plan -->
+        <div class="card">
+            <div class="card-body">
+                <div class="card-title mb-3">Report a problem</div>
+                <div class="card-subtitle mt-2 mb-2 pl-3 border-left border-warning" style="border-left-width: 5px !important;" >
+                    If an email cannot be delivered to your mailbox, please check <a href="/dashboard/notifications">your notifications</a> for error messages
+                </div>
+                <div class="mt-2">
+                    A support ticket will be created in Zendesk. Please do not include any sensitive information in the ticket.
+                </div>
+                <form id="supportZendeskForm" method="post" enctype="multipart/form-data">
+                    <div class="mt-4 mb-5">
+                        <label for="issueDescription" class="form-label">What happened?</label>
+                        <textarea class="form-control" name="ticketContents" id="issueDescription" rows="3" placeholder="Please provide as much information as possible. For example which alias(es), mailbox(es) ar affected, if this is a persistent issue...">{{- ticketContents or '' -}}</textarea>
+                    </div>
+                    <div class="mt-5 font-weight-bold">
+                        Attach files to support request
+                    </div>
+                    <div class="mt-1 text-muted">Only images, text and emails are accepted</div>
+                    <div class="custom-file mt-2">
+                        <input type="file" class="custom-file-input" name="ticketFiles" id="ticketFileGroup" multiple="multiple">
```

Comment:
Do you know if we can create a ticket without attachments?

Reply:
Yes, if the user doesn't upload any attachment, the ticket won't have any.